tests/core/test_template_tags.py

```diff
@@ -141,7 +137,6 @@ def test_obj_in__cache_and_articleforobjec_is_exist(self):
 
 # TODO manage plugins in template
 class WikiRenderTest(TemplateTestCase):
-
```

Comment:
random comment... but it's a bit weird how ruff claims to be compliant and then in some many cases removes non-harmful linebreak that Black didn't care about.

Reply:
Yeah, there might be a RULE for that... but I'm still not sure why its generating that diff

Reply:
I can check more deep just in case and revert as well all the changes to avoid this diff.

Reply:
I've been starring myself blind on this page: https://docs.astral.sh/ruff/settings/

I think there isn't a setting to prevent this behavior and I'm not sure where it's from. I think we can live with this in order to switch to ruff :+1: